rero_ils/modules/patron_transactions/listener.py

```diff
@@ -20,17 +20,17 @@
 from .api import PatronTransaction, PatronTransactionsSearch
 
 
-def enrich_patron_transaction_data(
-        sender, json=None, record=None, index=None, doc_type=None, **
-        dummy_kwargs):
+def enrich_patron_transaction_data(sender, json=None, record=None, index=None,
+                                   doc_type=None, arguments=None,
```

Comment:
In my point of view, as you never used `arguments` argument in any enrich function, it could be removed because it will be covered by `**dummy_kwargs` argument.

Reply:
This is the definition from https://github.com/inveniosoftware/invenio-indexer/blob/master/invenio_indexer/signals.py
```
before_record_index = _with_dynamic_connect(
    _signals.signal('before-record-index'))
"""Signal sent before a record is indexed.
The sender is the current Flask application, and two keyword arguments are
provided:
- ``json``: The dumped record dictionary which can be modified.
- ``record``: The record being indexed.
- ``index``: The index in which the record will be indexed.
- ``doc_type``: The doc_type for the record.
- ``arguments``: The arguments to pass to Elasticsearch for indexing.
- ``**kwargs``: Extra arguments.
```

I added it so we can see it exist.

Reply:
ok 👍